wls-wahlvorstand-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorstandservice/service/wahlvorstand/FunktionsnamenMapping.java

```diff
@@ -0,0 +1,14 @@
+package de.muenchen.oss.wahllokalsystem.wahlvorstandservice.service.wahlvorstand;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class FunktionsnamenMapping {
```

Comment:
Idee: Du könntest noch den Suffix `Properties` ergänzen um deutlich zu machen dass es eine Klasse nur für die Props ist.

Reply:
siehe commit [4e08898](https://github.com/it-at-m/Wahllokalsystem/pull/533/commits/4e08898f8d13750c68d13408257617215c461a1c)